addons/l10n_pl/data/account_tax_report_data.xml

```diff
@@ -172,22 +172,22 @@
         <field name="name">Podatek - Do przeniesienia</field>
         <field name="report_id" ref="tax_report"/>
         <field name="sequence" eval="3"/>
-        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 + PLTAXD_02_42</field>
+        <field name="formula">PLTAXC - PLTAXD</field>
     </record>
 
     <record id="account_tax_report_line_nad_naleznym" model="account.tax.report.line">
         <field name="name">Podatek - Nadwyżka naliczonego nad należnym</field>
         <field name="report_id" ref="tax_report"/>
         <field name="sequence" eval="1"/>
-        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 + PLTAXD_02_42</field>
+        <field name="formula">PLTAXC - PLTAXD</field>
```

Comment:
Isn't leaving the formula empty (so, with an explicit eval="False") summing all the children of the line already ? Can't we do that instead of repeating the formula ?

Reply:
No, we want a difference between C and D, not the sum.